src/screens/FeedScreen/Content.js

```diff
@@ -32,7 +33,8 @@ const Content = ({
   topics = [],
   item,
   onNewPollFetched,
-  setHaveSeeMore
+  setHaveSeeMore,
+  hasComment
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new prop `hasComment` has been added but there is no default value provided. This could potentially lead to unexpected behavior if the prop is not passed. It's a good practice to provide default values for optional props.
```diff
-  hasComment
+  hasComment = false
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->